### ISSUE TITLE
private-bower (1.0.1 and 1.0.2 )installation fails/hangs

### ISSUE THREAD

**Issue by igorlino** [Hacklone/private-bower#130]
Trying 
```
"npm install -g private-bower" 
```
will hang up. (so, 1.0.1 and 1.0.2 version are not installable)

The output from "C:\projects\ce-invoicecheck-gui>npm install -g private-bower --verbose" is:
```
C:\projects\ce-invoicecheck-gui>npm install -g private-bower --verbose
npm info it worked if it ends with ok
npm verb cli [ 'node',
npm verb cli   'C:\\Users\\icampista\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
npm verb cli   'install',
npm verb cli   '-g',
npm verb cli   'private-bower',
npm verb cli   '--verbose' ]
npm info using npm@2.8.4
npm info using node@v0.12.2
npm verb install initial load of C:\Users\icampista\AppData\Roaming\npm\package.json
npm verb readDependencies loading dependencies from C:\Users\icampista\AppData\Roaming\npm\package.json
npm verb cache add spec private-bower
npm verb addNamed "*" is a valid semver range for private-bower
npm verb addNameRange registry:http://registry.npmjs.org/private-bower not in flight; fetching
npm verb request uri http://registry.npmjs.org/private-bower
npm verb request no auth needed
npm info attempt registry request try #1 at 14:27:33
npm verb request id 313195132f051d89
npm verb etag "6BAET40P8AI718BG5E1MRTQYN"
npm http request GET http://registry.npmjs.org/private-bower
npm http 304 http://registry.npmjs.org/private-bower
npm verb etag http://registry.npmjs.org/private-bower from cache
npm verb get saving private-bower to C:\Users\icampista\AppData\Roaming\npm-cache\registry.npmjs.org\private-bower\.cache.json
npm verb addNamed "1.0.2" is a plain semver version for private-bower
npm verb afterAdd C:\Users\icampista\AppData\Roaming\npm-cache\private-bower\1.0.2\package\package.json not in flight; writing
npm verb afterAdd C:\Users\icampista\AppData\Roaming\npm-cache\private-bower\1.0.2\package\package.json written
npm info install private-bower@1.0.2 into C:\Users\icampista\AppData\Roaming\npm
npm info installOne private-bower@1.0.2
npm verb installOne of private-bower to C:\Users\icampista\AppData\Roaming\npm not in flight; installing
npm verb lock using C:\Users\icampista\AppData\Roaming\npm-cache\_locks\private-bower-a9997fa3bd79b5d0.lock for C:\Users\icampista\AppData\Roaming\npm\node_modules\private-bower
npm verb unbuild node_modules\private-bower
npm info preuninstall private-bower@0.6.5
npm info uninstall private-bower@0.6.5
npm verb unbuild rmStuff private-bower@0.6.5 from C:\Users\icampista\AppData\Roaming\npm\node_modules
npm verb gentlyRm vacuuming from C:\Users\icampista\AppData\Roaming\npm\private-bower.cmd up to C:\Users\icampista\AppData\Roaming\npm
npm verb gentlyRm vacuuming from C:\Users\icampista\AppData\Roaming\npm\private-bower up to C:\Users\icampista\AppData\Roaming\npm
npm info postuninstall private-bower@0.6.5
npm verb gentlyRm don't care about contents; nuking C:\Users\icampista\AppData\Roaming\npm\node_modules\private-bower
npm verb tar unpack C:\Users\icampista\AppData\Roaming\npm-cache\private-bower\1.0.2\package.tgz
npm verb tar unpacking to C:\Users\icampista\AppData\Roaming\npm\node_modules\private-bower
npm verb gentlyRm don't care about contents; nuking C:\Users\icampista\AppData\Roaming\npm\node_modules\private-bower
```
 
if i use an older version like .0.6.5  then the installation works:

```
npm install -g private-bower@0.6.5 --verbose
```

**Comment by igorlino** [comment]
Environment is: windows 7  64bits

```
C:\projects\ce-invoicecheck-gui>npm -v
2.8.4
```

```
C:\projects\ce-invoicecheck-gui>node -v
v0.12.2
```

```
C:\projects\ce-invoicecheck-gui>where npm
C:\Users\icampista\AppData\Roaming\npm\npm
C:\Users\icampista\AppData\Roaming\npm\npm.cmd
```

**Comment by jacobboland** [comment]
I noticed this as well, only on Windows. npm install is only partially installing the package.

My workaround was to download the release and extract it into node_modules. I then had to manually create the cmd file under npm. This works, but I get another issue https://github.com/Hacklone/private-bower/issues/132 when trying to run it.

**Comment by jacobboland** [comment]
Also, I tried cloning this repo in windows and it seems that master is in a weird state without me making any changes. It has a bunch of unstaged changes, deleting the files that are missing in the package.

**Comment by Hacklone** [comment]
This is fixed in 1.0.3 tried on Windows and it works for me.

**Comment by igorlino** [comment]
Confirmed. Install fine now, thanks. (there is a further issue, not related, i will raise it now)
